playground/docker-compose.local.yaml

```diff
@@ -87,10 +101,13 @@ services:
       CACHE_TYPE: remote
       CACHE_ADDRESS: redis:6379
       SERVER_PORT: 8090
+      PLAYGROUND_MITM_SERVICE_HOST: mitmproxy
+      PLAYGROUND_MITM_SERVICE_PORT: 8080
     ports:
       - "8090:8090"
     depends_on:
       - redis
+      - mitmproxy
 
   frontend:
     image: apache/beam_playground-frontend
```

Comment:
Why do we have to maintain both docker compose and kubernetes manifests? What if there is drift from these two artifacts that would lead to errors in production?  Why can't we have a single source of truth and someone just setup minikube on their local workstation and deploy the kubernetes manifests instead of docker compose?

Reply:
We're updating the playground to implement MiTM proxy as agreed in the meeting.
This approach sounds like a design change that will impact development and scope. If you would like let's discuss this in the coming meeting.